game/src/Game/GameStart.gd

```diff
@@ -107,9 +107,16 @@ func _setup_compatibility_mode_paths() -> void:
 func _load_compatibility_mode() -> void:
 	if GameSingleton.set_compatibility_mode_roots(_compatibility_path_list) != OK:
 		push_error("Errors setting game roots!")
-
+	
+	#CursorSingleton.load_cursors()
+	CursorManager.set_prefered_res(Vector2i(48,48))
+	CursorManager.load_cursors()
+	CursorManager.set_compat_cursor("normal",Input.CURSOR_ARROW)
```

Comment:
The cursor names should probably be StringNames, both as literals (`&"normal"`) and the variable/argument types that eventually are used to call `CursorSingleton` functions which accept a `StringName`

Reply:
Made this change.